mock/mock.go

```diff
@@ -252,18 +252,16 @@ func (m *Mock) findClosestCall(method string, arguments ...interface{}) (bool, *
 	return true, closestCall
 }
 
-func callString(method string, arguments Arguments, includeArgumentValues bool) string {
-
-	var argValsString string
-	if includeArgumentValues {
-		var argVals []string
-		for argIndex, arg := range arguments {
-			argVals = append(argVals, fmt.Sprintf("%d: %#v", argIndex, arg))
+func callString(method string, arguments Arguments) string {
+	var argVals []string
+	for i, arg := range arguments {
+		if i > 0 {
+			argVals = append(argVals, ",")
 		}
-		argValsString = fmt.Sprintf("\n\t\t%s", strings.Join(argVals, "\n\t\t"))
+		argVals = append(argVals, fmt.Sprintf("%#v", arg))
```

Comment:
But the nice feature of `%v` is that it works for pretty much all types, not only `[]byte` or `string`.

I also think that using Go's default formatting for types is more consistent than trying to come up with new formats, and it makes for a more "expected" result, even if it's a bit verbose.

A good alternative might be to shorten the `argVals` so a big `[]byte` is not completely printed, but limited to e.g 60 or so characters. This would mean seeing something like:
```
Expected: []byte{0x66, 0x6f, 0x6f, 0x66, 0x6f, 0x6f...
Received: []byte{0x62, 0x61, 0x72, 0x62, 0x61, 0x72...
```